src/transfer.ts

```diff
@@ -81,7 +82,12 @@ export class SimpleTxFlow extends BaseFlow {
         stepName: "execution",
         stepTimeoutMs: L2_EXECUTION_TIMEOUT,
         fn: async ({ recordStepGas, recordStepGasPrice, recordStepGasCost }) => {
-          const receipt = unwrap(await txResponse.wait(1));
+          let receipt;
+          if (this.l2EthersProvider != null) {
+            receipt = unwrap(await this.l2EthersProvider.waitForTransaction(txResponse.hash, 1));
+          } else {
+            receipt = unwrap(await txResponse.wait(1));
+          }
```

Comment:
Is this for lowering pool interval?

Reply:
from dm-s: needed due to .waitForTransaction from zksync-ethers not working with ZK OS receipts RPC response, specifly `l2ToL1Logs` field format